ingest/src/server_5g.rs

```diff
@@ -117,6 +116,7 @@ pub async fn grpc_server(shutdown: triggered::Listener, settings: &Settings) ->
         heartbeat_report_rx,
     )
     .deposits(Some(file_upload_tx.clone()))
+    .roll_time(Duration::minutes(15))
```

Comment:
nit: Might be better to move this to a const `INGEST_WAIT_DURATION` or similar?

Reply:
Maybe define it in `lib.rs` since its used in multiple places

Reply:
Fixed cbbeb29